client/loops.lua

```diff
@@ -1,5 +1,6 @@
 local statusInterval = require 'config.client'.statusIntervalSeconds
 local playerState = LocalPlayer.state
+local Active = false
```

Comment:
The `Active` variable isn't really necessary as `IsPauseMenuActive()` already returns a boolean value. Additionally, local variables should be camelCase.

Reply:
The variable is needed to capture the old state to understand if there is a change.